swagger.yml

```diff
@@ -672,6 +672,130 @@ paths:
           schema:
             $ref: "#/definitions/NotFound"
 
+  /groups/{groupID}/peers:
+    get:
+      operationId: getPeers
+      description: Get All VPC Peering Connections in One Project
```

Comment:
nit - might be worth mentioning we'll only fetch the first 💯 (https://docs.atlas.mongodb.com/reference/api/vpc-get-connections-list/#request-path-parameters)